.github/workflows/doc-build.yml

```diff
@@ -4,6 +4,11 @@
 name: Documentation GitHub Workflow
 
 on:
+  schedule:
```

Comment:
do not understand why this is needed.You need to explain why do we need another job building the docs on schedule in the main branch, we have this already.

Reply:
The publish workflow has been refactored to use the workflow_run event. It doesn't build anymore, just publishes from imported artifacts. This way we have a single workflow that is responsible for building docs and so we remove duplicates. A working example: https://github.com/nrfconnect/sdk-nrf/blob/master/.github/workflows/docpublish.yml